Lombiq.Tests.UI/Services/SqlServerManager.cs

```diff
@@ -139,9 +151,18 @@ public SqlServerRunningContext CreateDatabase()
 
         var destination = new BackupDeviceItem(filePathRemote, DeviceType.File);
         backup.Devices.Add(destination);
-        // We could use SqlBackupAsync() too but that's not Task-based async, we'd need to subscribe to an event which
-        // is messy.
-        backup.SqlBackup(server);
+        // We could use SqlBackupAsync() too but that's not Task-based async, we'd need to subscribe to an event
+        // which is messy.
+        await Task.Run(() => backup.SqlBackup(server));
```

Comment:
Why is this sync-in-async necessary?

Reply:
Because otherwise AsyncFixer will complain why we don't use `backup.SqlBackupAsync` instead.

Reply:
A suppression here would be better then.

I don't see why running this on a separate thread (if at all) would bring any benefits. We're blocking one thread in any case, and the rest of the code can't continue anyway.